app/src/main/java/com/example/pocketcloset/MainActivity.java

```diff
@@ -21,7 +21,7 @@ public class MainActivity extends AppCompatActivity {
     public BottomNavigationView bottomNavigation;
     CameraFragment cameraFragment = new CameraFragment();
     OutfitsFragment outfitsFragment = new OutfitsFragment();
-    WardrobeFragment wardrobeFragment = new WardrobeFragment();
+    WardrobeFragment wardrobeFragment = new WardrobeFragment(ParseUser.getCurrentUser());
```

Comment:
See my comment about not using Fragment constructors above. I would probably set this up in `onCreate`